resources/service.php

```diff
@@ -232,6 +232,18 @@
             'uri' => 'oauth/identity',
             'responseModel' => 'GetResponse',
         ],
+        'getProfile' => [
+	        'httpMethod' => 'GET',
+	        'uri' => 'users/{username}',
+	        'responseModel' => 'GetResponse',
+	        'parameters' => [
+		        'username' => [
+			        'type' => 'string',
+			        'location' => 'uri',
+			        'required' => true
+		        ],
+	        ],
+        ],
```

Comment:
Could you change the indentation level so it matches the rest of the file?

Reply:
oops, i should change indent settings, now it's good